doc/barman.1.d/50-list-backup.md

```diff
@@ -1,4 +1,4 @@
-list-backup *SERVER_NAME*
```

Comment:
I think it's worth renaming the file too so it matches the non-legacy command name.

(Edit: same applies to `50-list-server.md` and `50-show-server.md`)

Reply:
Good idea -- I've added a commit 👍